setup-gems-cache/action.yml

```
@@ -0,0 +1,32 @@
+name: Setup Gems Cache
+description: Setup Gems Caching for workflow
+inputs:
+  gems-path:
+    description: Homebrew's Ruby gems path
+    required: true
+  gems-hash:
+    description: Homebrew's Ruby Gemfile.lock sha256sum
+    required: true
+outputs:
+  gems-outcome:
```

Idk if `outcome` is a reserved keyword so this might not work, but I think I might prefer this if it's possible (but I don't feel strongly)

```suggestion
 outcome:
```

---

I think there are no reserved keyword on `steps.<step_id>.output.<output_name>` according to [this](https://docs.github.com/en/actions/learn-github-actions/contexts#steps-context).